.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: ruby
 sudo: false
+env:
+  global:
+    - secure: "gxjwgl9l9ZP/1t86odZyMzWyiVBkG1oFzeL+Kb6z9Dtfu954l9RbytlenTTcnRw1KtaDBZVBs0ix3K45b8T2S6JFzLQ88Z/36DmXGIN6vxctfnVD9sqbbyOuP++RhibK+qUqbPRHo6a3wBsMQ87DMj5S6pJe3dxbCKRXa8zcMd0="
+before_install:
+  - openssl aes-256-cbc -K $encrypted_df4ab1bff570_key -iv $encrypted_df4ab1bff570_iv -in .travis/secrets.tar.enc -out .travis/secrets.tar -d
```

Comment:
Turns out `before_deploy` is run _per deploy provider_, and running this setup twice will cause an error, as we discovered on `govuk_elements`. 

The most reliable way to limit decryption to situations where deployment is required is using `if [ "$TRAVIS_PULL_REQUEST_BRANCH" = "master" ];`, but this may not work well for multi-liners. See https://github.com/alphagov/govuk_elements/pull/415.